sources/packages/backend/workflow/test/models/assessment.model.ts

```diff
@@ -0,0 +1,177 @@
+import { AssessmentTriggerType, OfferingIntensity } from "@sims/sims-db";
+
+export enum YesNoOptions {
+  Yes = "yes",
+  No = "no",
+}
+
+export interface StudentDependentTable {
+  dateOfBirth: string;
+  attendingPostSecondarySchool: YesNoOptions;
+  declaredOnTaxes: YesNoOptions;
+}
+
+/**
+ * Offering delivery options.
+ */
+export enum OfferingDeliveryOptions {
+  Onsite = "onsite",
+  Online = "online",
+  Blended = "blended",
+}
+
+/**
+ * Canadian provinces.
+ */
+export enum Provinces {
```

Comment:
Should we move these ones to test helpers on lib?

Reply:
Thanks for moving the `Provinces` but I was also referring to the possibility to move the `YesNoOptions` and `OfferingDeliveryOptions` since they can be useful for others tests also.

Reply:
Was there a decision to not move `OfferingDeliveryOptions` as mentioned in the comment?